typescript/sdk/src/registry/GithubRegistry.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai';
+
+import { objLength } from '@hyperlane-xyz/utils';
+
+import { GithubRegistry } from './GithubRegistry.js';
+
+describe('GithubRegistry', () => {
```

Comment:
Note, since these make network requests, I'll put these with the new SDK E2E test group I'm setting up as part of my bigger const migration PR